api/v1alpha2/kubeadmbootstrapconfig_types.go

```diff
@@ -49,6 +49,9 @@ type KubeadmConfigStatus struct {
 }
 
 // +kubebuilder:object:root=true
+// +kubebuilder:resource:path=kubeadmconfigs,scope=Namespaced
+// +kubebuilder:storageversion
```

Comment:
do we have conversion enabled?

what does this do when conversion is not enabled?

Reply:
I think setting it proactively will help when we do enable conversions in the future

Reply:
The storageversion serves for conversion and for future version iterations. Other CAP* projects specify this field as well